docs/install_guides/installing-openvino-docker-windows.md

```diff
@@ -76,6 +76,62 @@ If you want to try some demos then run image with the root privileges (some addi
 docker run -itu ContainerAdministrator --rm <image_name> cmd /S /C "cd deployment_tools\demo && demo_security_barrier_camera.bat -d CPU -sample-options -no_show"
 ~~~
 
+## Build and Run the Docker* Image for GPU
+
+GPU Acceleration in Windows containers feature requires to meet Windows host, OpenVINO toolkit and Docker* requirements:
+* [Windows requirements](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/gpu-acceleration)
+  * The container host must be running Windows Server 2019 or Windows 10, version 1809 or newer.
+  * The container base image must be mcr.microsoft.com/windows:1809 or newer. Windows Server Core and Nano Server container images are not currently supported.
+  * The container host must be running Docker Engine 19.03 or newer.
+  * The container host must have a GPU running display drivers version WDDM 2.5 or newer.
+* [OpenVINO GPU requirement](https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_installing_openvino_windows.html#Install-GPU)
+  * Intel Graphics Driver for Windows version 15.65 or higher.
+* [Docker isolation mode requirement](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/hyperv-container)
+  *  Windows host and container version tags should be match.
+  *  [Windows host and container isolation process support](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility)
+
+##Build a Docker* Image for your host system
+
+1. You can re-use [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles). 
```

Comment:
Why do we say *can*? Is this step optional? Where else can the user get Dockerfiles?

Reply:
He can create himself Dockerfile or reuse our (it is prefer).

Reply:
Okay let's then have it like this:
```
1. Reuse one of [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles). You can also use your own Dockerfile.
```